core.js

```diff
@@ -56,6 +56,19 @@ function _check(buffer, headers, options) {
 	return true;
 }
 
+async function _checkSequence(sequence, tokenizer, options) {
+	options = {
+		position: 0,
+		...options
+	};
+
+	const buffer = Buffer.alloc(minimumBytes);
+
+	await tokenizer.peekBuffer(buffer, {position: options.position, length: 512, mayBeLess: true});
```

Comment:
`position: options.position` is the default, redundant.

Reply:
It isn't, as when used, I overwrite position to `1350` for the Adobe Illustrator check